packages/wonder-blocks-form/components/radio-group.md

```diff
@@ -22,6 +22,7 @@ class RadioGroupPokemonExample extends React.Component {
         this.state = {
             selectedValue: null,
         };
+        this.handleChange = this.handleChange.bind(this);
```

Comment:
I was always a little fuzzy as to whether you could use class properties when a constructor is present.

Reply:
I'm guessing in this case styleguidist's parser doesn't allow class properties in general.